core-common/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -159,4 +159,22 @@
      * @return strategy how to do null checking
      */
     NullValueCheckStrategy nullValueCheckStrategy() default ON_IMPLICIT_CONVERSION;
+
+    /**
+     * If MapStruct could not find another mapping method or apply an automatic conversion it will try to generate a
+     * sub-mapping method between the two beans. If this property is set to {@code true} MapStruct will not try to
+     * automatically generate sub-mapping methods.
+     * <p>
+     * If you decide to {@code disableSubMappingMethodsGeneration} please contact us at
```

Comment:
I would prefer to have this note only in the reference documentation, not the JavaDocs.